tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/SharedTaggedMetricRegistries.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.tritium.metrics.registry;
+
+import com.codahale.metrics.SharedMetricRegistries;
+import java.util.concurrent.TimeUnit;
+
+/** Analogous to codahale's {@link SharedMetricRegistries}. */
+public final class SharedTaggedMetricRegistries {
+
+    private static final TaggedMetricRegistry DEFAULT = new SlidingWindowTaggedMetricRegistry(1, TimeUnit.MINUTES);
```

Comment:
Before we release 0.12.0, let's double check whether 35 seconds was actually a better number for some reason...

Reply:
35 seconds is due to internal projects reporting metrics every 30 seconds. The though was to make the window as small as possible and still being confident we don't drop events - hence the 5 seconds margin.

Reply:
Ok so you're saying the danger is if we dump the metric registry on a slower frequency than its window then we lose information, but larger windows make each dump slightly more confusing (because it still contains data from a previous sample). 

Maybe we should just set the default to 35 seconds then and write some javadoc to this effect?

Reply:
Yes, if the window is smaller than our report frequency, we will lose information. For larger windows, I guess it won't actually be confusing, as the metrics will simply be aggregated over that window. It just seems desirable to have as granular measurements as possible, to be able to figure out when something changed.

Reply:
Adding some colour here since I introduced the 35 seconds aha. When we were doing our local benchmarks for timelock we were looking at extreme values i.e. max and p999 for a particularly small window i.e. 6-7 seconds, and having the window be at 60 seconds or even higher just threw our results off entirely for the entire window.

In theory it's probably fine having it be 30 seconds, I was just being conservative. Plus outside of repeated benchmarks, it's likely to not actually matter.

As an aside, do we necessarily want to encode WC metrics reporting frequency inside tritium? Granted it's probably difficult due to cjr but just curious.